terraform/monitoring/panels/app/publishing_workers_queued_size.libsonnet

```
@@ -0,0 +1,19 @@
+local grafana   = import '../../grafonnet-lib/grafana.libsonnet';
+local defaults  = import '../../grafonnet-lib/defaults.libsonnet';
+
+local panels    = grafana.panels;
+local targets   = grafana.targets;
+
+{
+  new(ds, vars)::
+    panels.timeseries(
+      title       = 'Messages queue size',
+      datasource  = ds.prometheus,
+    )
+    .configure(defaults.configuration.timeseries)
+    .addTarget(targets.prometheus(
+      datasource  = ds.prometheus,
+      expr        = 'sum(rate(publishing_queue_queued_size_total{}[$__rate_interval]))',
```

See current total, not the rate of the total changed?

```suggestion
      expr        = 'sum(publishing_queue_queued_size_total{}[$__rate_interval])',
```

---

Done.